src/Components/QuickGrid/Microsoft.AspNetCore.Components.QuickGrid/src/QuickGrid.razor.cs

```diff
@@ -387,7 +392,15 @@ private string AriaSortValue(ColumnBase<TGridItem> column)
         : ColumnClass(column);
 
     private string GridClass()
-        => $"quickgrid {Class} {(_pendingDataLoadCancellationTokenSource is null ? null : "loading")}";
+    {
+        var gridClass = $"quickgrid {Class} {(_pendingDataLoadCancellationTokenSource is null ? null : "loading")}";
+        if (AdditionalAttributes?.TryGetValue("class", out var originalClass) == true)
+        {
+            gridClass += $" {originalClass}";
+        }
```

Comment:
You can utilize the AttributeUtilities.CombineClassNames method instead: https://github.com/dotnet/aspnetcore/blob/33243ad0a057a1d99fd2b7fe0f9765d4b28d641d/src/Components/Web/src/Forms/AttributeUtilities.cs#L10

Here is a usage example in the `CssClass` property of the `InputBase` type; https://github.com/dotnet/aspnetcore/blob/33243ad0a057a1d99fd2b7fe0f9765d4b28d641d/src/Components/Web/src/Forms/InputBase.cs#L187

Reply:
Thank you @mkArtakMSFT , I have done the changes, but AttributeUtilities.CombineClassNames is internal that can't be accessed in QuickGrid library. How to solve that?